Documentation/openshiftdocs/modules/remotewritespec.adoc

```diff
@@ -35,7 +35,7 @@ link:prometheusrestrictedconfig.adoc[PrometheusRestrictedConfig]
 
 |oauth2|*monv1.OAuth2|Defines OAuth2 authentication settings for the remote write endpoint.
 
-|proxyUrl|string|Defines an optional proxy URL. It is superseded by the cluster-wide proxy, if enabled.
+|proxyUrl|string|Defines an optional proxy URL. It is superseded by the cluster-wide proxy when it is enabled, prioritizing the HTTPS proxy.
```

Comment:
cc @eromanova97, if there is any better wording for this.
(when cluster-wide proxy is enabled, it can have an HTTP and an HTTPS proxy and we make HTTPS take precedence)